src/App/Pages/Send/SendGroupingsPage/SendGroupingsPageListItem.cs

```diff
@@ -66,5 +66,31 @@ public string Icon
                 return _icon;
             }
         }
+
+        public string AutomationId
+        {
+            get
+            {
+                if (_name != null)
+                {
+                    return "SendItem";
+                }
+                if (Type != null)
+                {
+                    switch (Type.Value)
+                    {
+                        case SendType.Text:
+                            return "SendTextFilter";
+                            break;
+                        case SendType.File:
+                            return "SendFileFilter";
+                            break;
+                        default:
+                            break;
```

Comment:
⛏️ `break;` is not needed after `return ...` in a `case` and I think you can remove the `default` as well

Reply:
You are right! Updating it...